js/app/modules/contentGroup/contentGroup.js

```diff
@@ -92,6 +93,7 @@ const ContentGroup = new Module.Class({
 
                 // Style class needs to go on the button itself, so that we get :hover states and the like.
                 title_button.get_style_context().add_class(Utils.get_element_style_class(ContentGroup, 'title'));
+                title_button.get_style_context().add_class(Utils.get_element_style_class(ContentGroup, 'title_clickableStyle'));
```

Comment:
yeah I was tihnking about this. you're right. I'll separate them into 2 different styles. `ContentGroup__title` and `ContentGroup__title--clickable`.